src/Development/IDE/Types/Exports.hs

```diff
@@ -16,8 +18,17 @@ import Name
 import FieldLabel (flSelector)
 import qualified Data.HashMap.Strict as Map
 import GhcPlugins (IfaceExport)
+import Data.HashSet (HashSet)
+import qualified Data.HashSet as Set
+import Data.Bifunctor (Bifunctor(second))
```

Comment:
I would usually use Data.Tuple.Extra.second here

Reply:
I'm ambivalent. Leaving as it is to skip 3h waiting for CI